### PR TITLE
update traslation: safely-drain-node

### PR DESCRIPTION
**Pull request by lianghao208**
The english version of the doc [_safely-drain-node_ has been](https://github.com/kubernetes/website/commit/720812a0ce96bbc9115094370d1c0a322ee8a91b) updated. So I update the translation as well.

### CODE REVIEW

File: content/zh/docs/tasks/administer-cluster/safely-drain-node.md [hunk 1]
```diff
@@ -231,19 +232,27 @@ For a given eviction request, there are two cases:
...
+
+For example: this can happen if ReplicaSet is creating Pods for your application but
+the replacement Pods do not become `Ready`. You can also see similar symptoms if the
+last Pod evicted has a very long termination grace period.
 
 In this case, there are two potential solutions:
 
-- Abort or pause the automated operation. Investigate the reason for the stuck application, and restart the automation.
-- After a suitably long wait, `DELETE` the pod instead of using the eviction API.
+- Abort or pause the automated operation. Investigate the reason for the stuck application,
```
Comment (tanjunchen) [inline content/zh/docs/tasks/administer-cluster/safely-drain-node.md:246]: could you sync there changes ?

Comment (lianghao208) [reply content/zh/docs/tasks/administer-cluster/safely-drain-node.md:246]: ok, I will sync those changes.

Comment (lianghao208) [reply content/zh/docs/tasks/administer-cluster/safely-drain-node.md:246]: I've synced the changes. @tanjunchen  Thanks for the review.

**Review by tanjunchen — COMMENTED** [review]
@lianghao208
/approve
/lgtm

**Pull request merged**